src/app/core/components/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,8 +1,25 @@
 import { Component } from '@angular/core';
 
+import { Observable } from 'rxjs';
+
+import { BreadcrumbsStep } from '@app/interfaces/helpers/breadcrumbs-step.interface';
```

Comment:
interfaces for core services, I believe, might be stored within `core` models/interfaces. Just place services and related interfaces near each other.

Reply:
Okay